### PR TITLE
Added forwarder project for Kernel32::VerifyVersionInfoA

### PR DESCRIPTION
**Pull request by zfields**
DO NOT PULL - FOR REVIEW ONLY

### CODE REVIEW

File: User32/Kernel32/Kernel32.vcxproj [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
+    <LinkIncremental>false</LinkIncremental>
+  </PropertyGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
+    <ClCompile>
+      <PrecompiledHeader>Use</PrecompiledHeader>
+      <WarningLevel>Level3</WarningLevel>
+      <Optimization>Disabled</Optimization>
+      <PreprocessorDefinitions>WIN32;_DEBUG;_WINDOWS;_USRDLL;KERNEL32_EXPORTS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```
Comment (zfields) [inline User32/Kernel32/Kernel32.vcxproj]: I changed USER32_EXPORTS to KERNEL32_EXPORTS, but I'm not sure what this corresponds with...

File: User32/Kernel32/stdafx.h [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+// are changed infrequently
+//
+
+#pragma once
+
+#include "targetver.h"
+
+#define WIN32_LEAN_AND_MEAN             // Exclude rarely-used stuff from Windows headers
+// Un-#define WINUSERAPII as it normally means declspec(dllimport) but we're providing implementations instead
+#undef WINUSERAPI
```
Comment (ooeygui) [inline User32/Kernel32/stdafx.h]: You probably don't need WINUSERAPI

File: User32/Kernel32/stubs.cpp [hunk 3]
```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Open Technologies, Inc.  All rights reserved.  
+// Licensed under the BSD 2-Clause License.  
+// See License.txt in the project root for license information.
+
+#include "stdafx.h"
+
+WINUSERAPI
```
Comment (ooeygui) [inline User32/Kernel32/stubs.cpp]: Is this how it is declared in kernel32?

File: User32/Kernel32/stubs.cpp [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+    VersionInfoW.dwMajorVersion = VersionInfo->dwMajorVersion;
+    VersionInfoW.dwMinorVersion = VersionInfo->dwMinorVersion;
+    VersionInfoW.dwBuildNumber = VersionInfo->dwBuildNumber;
+    VersionInfoW.dwPlatformId = VersionInfo->dwPlatformId;
+    VersionInfoW.wServicePackMajor = VersionInfo->wServicePackMajor;
+    VersionInfoW.wServicePackMinor = VersionInfo->wServicePackMinor;
+    VersionInfoW.wSuiteMask = VersionInfo->wSuiteMask;
+    VersionInfoW.wProductType = VersionInfo->wProductType;
+    VersionInfoW.wReserved = VersionInfo->wReserved;
```
Comment (ooeygui) [inline User32/Kernel32/stubs.cpp]: You'll need to translate it back to ansi. That includes the strings. Remember VersionInfo is an in-out parameter.

**Comment by ooeygui** [comment]
closing the request based on our conversation. Submit a new one when ready

**Pull request closed without merging**